### ISSUE TITLE
`id` of type UUID with Primary Key does not reflect `->primary()` in migration.

### ISSUE THREAD

**Issue by 949mac** [TiagoSilvaPereira/vemto-issues#183]
Actual
```
$table->uuid('id');
```

Expected
```
$table->uuid('id')->primary();
```

*Showing Vemto Setup*
- [x] id is `UUID`
- [x] id is Primary Key

![image](https://user-images.githubusercontent.com/23151962/124033999-7fb4ed80-d9af-11eb-96f7-f03cea4fbbfd.png)

*Showing produced migration*
- [ ] `id` is missing `->primary()`

![image](https://user-images.githubusercontent.com/23151962/124034011-85aace80-d9af-11eb-80fe-975f8eecfe8f.png)

*Showing error in DB*
![image](https://user-images.githubusercontent.com/23151962/124034205-c6a2e300-d9af-11eb-8161-6f3fbfb7ca03.png)

**Comment by 949mac** [comment]
Quick little Mac hack if anyone runs into this before a proper patch:

(run this from the terminal of the project root)

```bash
grep -rl "table->uuid('id');" ./database/migrations | xargs sed -i '' "s/table->uuid('id');/table->uuid('id')->primary();/g"
```

**Comment by TiagoSilvaPereira** [comment]
Hi @949mac thank you for reporting, I'll fix it asap

**Comment by TiagoSilvaPereira** [comment]
Fixed on 0.9.8
